technology/kotlin-quarkus/src/main/kotlin/org/acme/kotlin/schooltimetabling/rest/TimetableResource.kt

```diff
@@ -0,0 +1,255 @@
+package org.acme.kotlin.schooltimetabling.rest
+
+import ai.timefold.solver.core.api.score.analysis.ScoreAnalysis
+import ai.timefold.solver.core.api.score.buildin.hardsoft.HardSoftScore
+import ai.timefold.solver.core.api.solver.ScoreAnalysisFetchPolicy
+import ai.timefold.solver.core.api.solver.SolutionManager
+import ai.timefold.solver.core.api.solver.SolverManager
+import jakarta.inject.Inject
+import jakarta.ws.rs.*
```

Comment:
Please configure your IDE to never use `*` imports.
(One of our little rules; makes reviews easier, and creates less classloading work for the JVM.)